src/requirements.txt

```diff
@@ -30,3 +30,6 @@ wincertstore==0.2
 yarl==1.7.2
 zope.event==4.5.0
 zope.interface==5.4.0
+os 
```

Comment:
make sure to use pip freeze > requirements.txt so that it includes the version we are currently using